tests/sles4sap/wizard_hana_install.pm

```diff
@@ -24,8 +24,8 @@ sub run {
 
     my $timeout  = bmwqemu::scale_timeout(3600);
     my $sid      = get_required_var('INSTANCE_SID');
-    my $password = 'Qwerty_123';
-    set_var('PASSWORD', $password);
```

Comment:
Same comment as in `tests/sles4sap/hana_install`: the `PASSWORD` variable is further used by `tests/sles4sap/hana_test` and `tests/sles4sap/hana_cluster`, so either keep the `set_var('PASSWORD', $password)` in this module, or modify also `tests/sles4sap/hana_cluster` so it does not look for the password in the `PASSWORD` variable.